src/lm_buddy/cli/run.py

```diff
@@ -11,27 +18,19 @@ def group():
 @group.command("simple", help="Run the simple test job.")
 @click.option("--config", type=str)
 def run_simple(config: str) -> None:
```

Comment:
this is a nice refactor to include imports at the top-level, makes it much cleaer what's being run

Reply:
Thanks! Although reminder that the internal imports were previously there for a purpose (to allow for executign portions of the library depending on which optional group was installed).

But the optional groups felt kind of silly after all given that the dependency set between eval/finetuning is roughhyl equivalent. So now that its removed, we have no reason to keep the internal imports.